File: redshift.php
<?php
   $user = "awsuser";
   $password = "Americaniche1";
   $ODBCConnection = odbc_connect("DRIVER={Amazon Redshift (x64)}; Server=redshift-cluster-1.cpmrep2pkzz9.us-east-1.redshift.amazonaws.com; Database=dev;Port=5349;String Types=Unicode", $user, $password);
   $SQLQuery = "SELECT * FROM cashflow_originator";
   echo "$SQLQuery";
   $RecordSet = odbc_exec($ODBCConnection, $SQLQuery);
   while (odbc_fetch_row($RecordSet)) {
        $result = odbc_result_all($RecordSet, "border=1");
        echo $result;
    }
    odbc_close($ODBCConnection);
   ?>

File: redshift.py
import psycopg2
def redshift():
    conn = psycopg2.connect(dbname='dev', host='redshift-cluster-1.cpmrep2pkzz9.us-east-1.redshift.amazonaws.com', port='5439', user='awsuser', password='Americaniche1')
    cur = conn.cursor()
    cur.execute("SELECT * FROM `Cashflow`;")
    cur.fetchall()
    cur.close()
    conn.close()

redshift();
